longevity_large_partition_test.py

```diff
@@ -19,3 +19,4 @@ def pre_create_large_partitions_schema(self, compaction_strategy=CompactionStrat
                     CREATE KEYSPACE IF NOT EXISTS scylla_bench WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3}
             """)
             session.execute(create_table_query)
+            session.execute(create_mv_on_scylla_bench_table_query())
```

Comment:
Why it was moved from configuration to here?

Reply:
Roy asked to add it to all large partition jobs